x-pack/filebeat/module/zeek/dns/config/dns.yml

```diff
@@ -36,3 +36,9 @@ processors:
           
       ignore_missing: true
       fail_on_error: false
+{{ if .community_id }}
+  - community_id:
+      fields:
+        source_ip: source.address
+        destination_ip: destination.address
```

Comment:
How does the `community_id` processor react, when getting domain names in these IP parameters? I'd rather have this be based on destination/source.ip and skip the processor when they're missing

Reply:
If the values are not IP addresses then the processor will just move on and do nothing.

It's based on .address because that is what is populated by the beat rename processor. The source.ip and destination.ip get populated later by the ingest node pipeline.